frontend/src/types/analytics.ts

```diff
@@ -23,6 +25,19 @@ export type Events = {
     /** the identifier of the image */
     identifier: string
   }
+  /**
+   * Description: The user clicks the CTA button to the external source to use the image
+   * Questions:
+   *   - How often do users go to the source after viewing a result?
+   */
+  GET_MEDIA: {
+    /** The unique ID of the media */
```

Comment:
Nit: since these aren't sentences...

```suggestion
    /** the unique ID of the media */
```